style.css

```diff
@@ -1,31 +1,39 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter');
 
-* {
-  background-color: #e5e5e5;
-}
-
 header {
+  background-color: azure;
```

Comment:
- [ ] **REQUIRED:** Please follow the design given on **Figma**, it is actually white, and for your page the different color making the UI/UX a bit ugly.
![image](https://user-images.githubusercontent.com/77872355/129299077-3332edd3-cb3c-4dc8-9220-c1fc94850890.png)

Reply:
I did it like that so that I have a sticky navbar on top, but I removed it now

Reply:
But as you can see the colour is not covering full-width, and it is better to follow the design because UI/UX specialists created these templates according to modern software demand.